scripts/m15-check.sh

```
@@ -0,0 +1,118 @@
+
+ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
+cd "$ROOT"
+
+BENCH_DIR=${RC_BENCHMARK_DIR:-}
+SOAK_DIR=${RC_SOAK_DIR:-}
+RELEASE_DIR=${RC_RELEASE_DIR:-}
+STAGING_FILE=${STAGING_EVIDENCE_FILE:-}
+WAIVER_FILE=${RC_8H_WAIVER:-}
+
+require_dir_var() {
+  local name=$1 value=${!name:-}
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)